.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -256,31 +319,181 @@ demisto-sdk-nightly:run-end-to-end-tests:
+    - echo "SemperisDSP/ModelingRules/SemperisDSP" >> "${ARTIFACTS_FOLDER_INSTANCE}/modeling_rules_to_test.txt"
+    - cat "${ARTIFACTS_FOLDER_INSTANCE}/modeling_rules_to_test.txt"
+    - section_end "Update Packs To Install And Modeling Rules To Test"
+
+    - section_start "Create Artifacts Instance Directory" --collapsed
+    - |
+      echo "Creating Artifacts instance directory: ${ARTIFACTS_FOLDER_INSTANCE}"
+      [ ! -d "${ARTIFACTS_FOLDER_INSTANCE}" ] && mkdir -p "${ARTIFACTS_FOLDER_INSTANCE}"
+      echo "${INSTANCE_ROLE}" > "${ARTIFACTS_FOLDER_INSTANCE}/instance_role.txt"
+    - section_end "Create Artifacts Instance Directory"
+
+    - section_start "Copy Env Results To Artifacts Folder" --collapsed
+    - |
+      # workaround for the hard-coded value in the sdk
+      cp "${ARTIFACTS_FOLDER_SERVER_TYPE}/filter_file.txt" "./artifacts/filter_file.txt"
+    - section_end "Copy Env Results To Artifacts Folder"
+
+    - !reference [.download-demisto-conf]
+    - !reference [.secrets-fetch]
+    - !reference [.lock-machine]
+
+    - !reference [.export_cloud_machine_constants]
+    - echo "Server URL $DEMISTO_BASE_URL"
+    - echo "XSIAM_AUTH_ID $XSIAM_AUTH_ID"
+
+    - section_start "Test Modeling Rules" --collapsed
+    - demisto-sdk upload -z -x -i Packs/SemperisDSP --insecure
+    - ./Tests/scripts/test_modeling_rules.sh || EXIT_CODE=$?
+    - section_end "Test Modeling Rules"
+
+    - section_start "End to End Tests" --collapsed
+    - |
+      mkdir $ARTIFACTS_FOLDER/demisto-sdk
+      git clone -b ${SDK_REF} --single-branch --depth 1 https://github.com/demisto/demisto-sdk.git $ARTIFACTS_FOLDER/demisto-sdk
+      echo "Server URL: $DEMISTO_BASE_URL"
+      python3 -m pytest $ARTIFACTS_FOLDER/demisto-sdk/tests_end_to_end/xsiam || EXIT_CODE=$?
+    - section_end "End to End Tests"
+
+    - section_start "Install Packs" --collapsed
+    - cat "$ARTIFACTS_FOLDER_SERVER_TYPE/filter_file.txt" 
+    - ./Tests/scripts/install_content_and_test_integrations.sh || EXIT_CODE=$?
+    - cp -f "${ARTIFACTS_FOLDER_SERVER_TYPE}/conf.json" Tests/conf.json
+    - section_end "Install Packs"
+
+    - !reference [.run_test_content]
+
+    - section_start "Cleanup env results from artifacts folder" --collapsed
+    - |
+      # workaround for the hard-coded value in the sdk
+      rm -f "./artifacts/filter_file.txt"
+    - section_end "Cleanup env results from artifacts folder"
+
+    - !reference [.cloud-machine-information]
+    - exit $EXIT_CODE
+
+  after_script:
+    - source .gitlab/helper_functions.sh
+    - !reference [.unlock-machine]
+
+demisto-sdk-nightly:run-end-to-end-tests-xsoar-ng:
+  tags:
+    - gce
+  extends:
+    - .default-job-settings
+    - .sdk-nightly-schedule-rule-always
+  services: # needed for running e2e tests for docker
+    - name: docker.art.code.pan.run/build-tools--image-dind:20.10.12-dind
+      alias: docker
+  variables:
+    CLOUD_MACHINES_TYPE: "build"
+    CLOUD_MACHINES_COUNT: 1
+    INSTANCE_ROLE: "XSOAR SAAS"
```

Comment:
```suggestion
    INSTANCE_ROLE: "xsoar_saas"
```

Reply:
Are you sure? the rest INSTANSE_ROLE are in all caps lock

Reply:
i'm sure

Reply:
So does that mean that I need to change the rest as well?